examples/case_studies/multilevel_modeling.myst.md

```diff
@@ -242,13 +242,13 @@ Let's now turn our attention to the unpooled model, and see how it fares in comp
 coords = {"county": mn_counties}
 
 with pm.Model(coords=coords) as unpooled_model:
-    floor_idx = pm.MutableData("floor_ind", floor_measure, dims="obs_id")
+    floor_idx = pm.MutableData("floor_idx", floor_measure, dims="obs_id")
```

Comment:
Oh, wait a minute, it was just called `"floor_ind"` but actually contains the data `floor_measure`, which I guess is different than the data `floor_ind` used below? Pretty confusing naming.

Reply:
It's called floor measure all the way through the notebook. I do find it confusing as to why floor and county are suffixed with `_idx` but contain the floor and county arrays.

Reply:
Confusing indeed, in the above model it's created like this:
`    floor_ind = pm.MutableData("floor_ind", floor_measure, dims="obs_id")`

and then used in the regression. let's just use the same convention here.